website/docs/r/front_door.html.markdown

```diff
@@ -184,11 +195,13 @@ The `routing_rule` block supports the following:
 
 The `forwarding_configuration` block supports the following:
 
-* `backend_pool_name` - (Required) The name of the Front Door Backend Pool. 
+* `backend_pool_name` - (Required) Specifies the name of the Backend Pool to forward the incoming traffic to. 
+
+* `cache_enabled` - (Optional) Specifies whether to Enable caching or not. Valid options are `true` or `false`. Defaults to `true`.
 
-* `cache_use_dynamic_compression` - (Optional) Whether to use dynamic compression when caching. Valid options are `true` or `false`. Defaults to `true`.
+* `cache_use_dynamic_compression` - (Optional) Whether to use dynamic compression when caching. Valid options are `true` or `false`. Defaults to `false`.
```

Comment:
should this be true?

Reply:
I believe this was a documentation error, as in the schema in master the default value is also false. So I think this is good.